src/Illuminate/Console/Application.php

```diff
@@ -55,11 +55,13 @@ public function __construct(Container $laravel, Dispatcher $events, $version)
      */
     public function call($command, array $parameters = [])
     {
-        $parameters['command'] = $command;
+        $parameters = collect($parameters)->prepend($command);
 
         $this->lastOutput = new BufferedOutput;
 
-        return $this->find($command)->run(new ArrayInput($parameters), $this->lastOutput);
+        $this->setCatchExceptions(false);
```

Comment:
> Because we are using the console application, it (as it should) has exception rendering turned on to display console friendly errors.
> But when calling via the facade the exception should NOT get supressed (or rendered in console friendly output) as the calling code needs to receive those errors thrown.

But isn't `php artisan foo` going to use the same method. `Illuminate\Console\Application::call()` will be executed from `Illuminate\Foundation\Console\Kernel::handle()` <https://github.com/laravel/framework/blob/5.1/src/Illuminate/Foundation/Console/Kernel.php#L102>

Which will be called from <https://github.com/laravel/laravel/blob/master/artisan#L33>.